### DISCUSSION TITLE
Configuring report type using global or project configuration

### DISCUSSION THREAD

**Discussion by chandrasekarsankaranarayanan-alation** [getgauge/gauge#2609] [General]
Hi,

 I would like to understand how report generation is controlled in `Gauge`. For my purpose, I want to generate both `html` and `xml` reports for test runs. However, only `xml` result is generated even though both the plugins are installed. Is there a way to control or configure this?

Thanks,
Chandra

**Comment by sriv** [comment]
Hi,

The reporting plugins are set per project in the `manifest.json` -for example https://github.com/getgauge/gauge-tests/blob/master/manifest.json -> this projects generates both html and xml report. You may check if your project has only xml-report plugin included int he manifest.

Hope this helps.

**Comment by chandrasekarsankaranarayanan-alation** [comment]
Hi sriv,

  Thanks for the response. I figured it out by trial and error. It will be really helpful if this information is included in official documentation, if not already available.

**Reply by chadlwilson** [reply]
The documentation is open source. if you can find the appropriate place you'd expect to find the info, please submit a PR.
